test/test_linearizer.py

```diff
@@ -108,6 +108,7 @@ def test_end_local(self):
     self.assertEqual(k.uops.uops[-1].uop, UOps.ENDIF)
     self.assertLess(k.uops.uops.index([x for x in k.uops.uops if x.uop is UOps.STORE][-1]), k.uops.uops.index(k.uops.uops[-1]))
 
+  @unittest.skipIf(getenv("PTX"), "Ptx adds additional uops for load indexing")
```

Comment:
instead of skipping, can you update tests to test PTX behaviors?

Reply:
Sure thing, updated